docs/section6/Rule6-7.md

```diff
@@ -12,9 +12,12 @@
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
 ## Rule Logic: 
+- List lighting space types that have no daylighting control requirements: `not_applicable_space_types = ["DORMITORY_LIVING_QUARTERS", "FIRE_STATION_SLEEPING_QUARTERS", "HEALTHCARE_FACILITY_OPERATING_ROOM", "OUTPATIENT_HEALTH_CARE_FACILITIES_CLASS_1_IMAGING_ROOMS", "DWELLING_UNIT", "GUEST_ROOM", "STORAGE_ROOM_SMALL", "PARKING_AREA_INTERIOR"]`
 
```

Comment:
@JacksonJ-KC I do not see this space type exception in G3.1.-6(h). What am I missing?

Reply:
SG understands why this was done. suggest adding reference to relevant table in Section 9 which states that daylight dimming controls are not required for those space types